### ISSUE TITLE
🚧 Dev ➾ Add Support for Public TZkT Instances to the VScE UI

### ISSUE THREAD

**Issue by hu3man** [ecadlabs/taqueria#1345]
### 🚥 Definition of Done

- [ ] 🎯 Planning and Design
- [ ] 🏎️ Dev Implementation (PR merged)
- [ ] ✍️ Release Notes Added
- [ ] 🏆 Product Acceptance

### ⚖️ Acceptance Criteria

✨ Functional: 
- [ ] The TZkT UI component in VScE can be pointed at a public TZkT instance (ghostnet indexer for example)
- [ ] The UI should only listen for contracts/operations known by Taqueria (populated from aliases/state)
- [ ] Replacing the use of polling with subscriptions
- [ ] Upon switching environments, the UI should switch to the correct TZkT source
 
 📐 Non-Functional: 
- [ ] ... 
 

### 🔭 Scope

🙅‍♂️ Out of Scope: 
- ... 
 
 

### 🧰 Work Description

Currently, the VScE TzKT indexer only works on a sandbox. We want to expand this so that the UI in VScE can show relevant contracts and operations on testnets as well

### 🦄 Sizing

7

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by jevonearth** [comment]
Notes from today's standup;

Some blockers around tzkt for @MichalLytek , @AlirezaHaghshenas are helping unblock that. Some or all of the filtering is implemented

Work on changing polling to subscriptions is underway. We need to talk about this when @hu3man is back in the fold.
